pages/History.qml

```diff
@@ -1510,7 +1527,7 @@ Rectangle {
             root.txModelData.push({
                 "i": i,
                 "isout": isout,
-                "amount": Number(amount),
+                "amount": Utils.removeTrailingZeros(displayAmount.toFixed(12)),
```

Comment:
Why this change is needed?

Reply:
I removed Number() because amounts like 0.000001 were returning 0.

Reply:
> I removed Number() because amounts like 0.000001 were returning 0.

How can i reproduce the issue?

Reply:
In this example, transactions #1 to #6 the wallet knows the amount. Transaction #7 the wallet doesn't know the amount (wallet was restored at this point).

`"amount": Utils.removeTrailingZeros(displayAmount.toFixed(12))`: all transactions with known amount  (#1-6) display the amount correctly. 
![image](https://user-images.githubusercontent.com/45968869/94722985-5d2a7100-0358-11eb-9a5a-4f83910a8bf9.png)


` "amount": Number(amount)`: some transactions (#3-6) with known amount display as Unknown amount. 
![image](https://user-images.githubusercontent.com/45968869/94722858-2eac9600-0358-11eb-8df9-b7e3c10d77dc.png)

Reply:
Can't reproduce. Works well with `"amount": Number(amount)` (Qt 5.9.7). What Qt version do you use?

Reply:
Did you send the txes in question to the same wallet but to the different account or did you send them to a third party wallet?

Reply:
I checked with Qt 5.12.9 and 5.15.1 and also can’t reproduce this issue. @rating89us can you double check?

Reply:
My wallet main address (account #1) is 48rg...
The first two transactions are from account #1 (48rg...) to account #2's primary address (89HP...) and its first subaddress (88PS...).
The remaining transactions are from account #1 to account #1 (48rg... to 48rg...)

I'm using Qt 5.12.9.
I'll check again if I can identify what's causing this.

Reply:
There is a comment in line 1510 :
```
// *sometimes* amount is 0, while the 'destinations string'
// has the correct amount, so we try to fetch it from that instead.
```

In my investigation "amount = 0" occurs when you send from account #1 to any address of account #1 (either main address or subaddress).

If I send from account #1 to account #2 (either main address or subaddress), the amount is correct.

Reply:
Reproduced the issue. Looking into it.

Reply:
https://github.com/monero-project/monero-gui/pull/3127 resolves the issue. Please confirm.

Reply:
Yes, it's working now. Thanks

Reply:
I won't fix this in this PR, since it will be fixed by your PR #3127 anyway